flowr/src/main/java/com/fueled/flowr/Flowr.java

```diff
@@ -282,15 +287,19 @@ protected final String getTagPrefix() {
                 transaction.add(mainContainerId, fragment);
             }
 
-            transaction.commit();
+            identifier = transaction.commit();
 
             if (data.isSkipBackStack()) {
                 setCurrentFragment(fragment);
             }
 
+            if (data.getTargetFragmentId() != null) {
+                targetFragmentId = data.getTargetFragmentId();
+            }
         } catch (Exception e) {
             Log.e(TAG, "Error while displaying fragment.", e);
         }
+        return identifier;
```

Comment:
Kindly add documentation on the newly returned value from the function.

Reply:
Done! Thanks 👍